README.rst

```diff
@@ -224,7 +224,9 @@ Should the need ever arise to purge the queue of jobs, run the following
 
 .. code:: bash
 
-    $ celery --app portal.celery_worker.celery purge
+    $ celery purge --app portal.celery_worker.celery
```

Comment:
Can you add `--force` here? otherwise yes/no will be prompted

Reply:
good point.